### DISCUSSION TITLE
How can I change the user specific directories?

### DISCUSSION THREAD

**Discussion by sioinfor** [prasathmani/tinyfilemanager#572] [Q&A]
I want to achive that each logged in user will see it's own folder. For example:
pdoks/User1
pdoks/User2
pdoks/User3

Config :
`$root_path = $_SERVER['DOCUMENT_ROOT'].'/pdoks';`
`$root_url = '/pdoks/`
`$directories_users = array(
 'User1' => 'pdoks/User1',
 'User2' => 'pdoks/User2',
 'User3' => 'pdoks/User3
);`

Now the user only sees its own folder, but the preview links, and the download links are not working.
If I delete the $directories_users, so the User1 logged in can see all user's folders, the links will work again.

Thanks for the help!

**Comment by prasathmani** [comment]
```PHP
$directories_users = array( 
'User1' => $_SERVER['DOCUMENT_ROOT'].'pdoks/User1', 
'User2' => $_SERVER['DOCUMENT_ROOT'].'pdoks/User2', 
'User3' => $_SERVER['DOCUMENT_ROOT'].'pdoks/User3';
);
```
